cypress/locators/validations/locators.js

```diff
@@ -0,0 +1,2 @@
+export const DEPRECATED_ERROR_MESSAGE = '.common-input__message--error';

```

Comment:
could we make `.common-input__` as a separate locator and after that re-use it here in those 2 locators? Probably move it to `common` locators?

Reply:
fixed